services/net/cpnews/appsettings.Development.json

```diff
@@ -0,0 +1,26 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "Service": {
+    "MaxFailLimit": 5,
+    "ApiUrl": "http://host.docker.internal:40010/api",
+    "MediaTypes": ["CPNews"]
+  },
+  "Auth": {
+    "Keycloak": {
+      "Authority": "http://host.docker.internal:40001/auth/realms/tno",
+      "Audience": "tno-service-account",
+      "Client": "tno-service-account",
```

Comment:
Checkout the latest changes in my prior PR for configuration.  There are some differences in the Keycloak section.

Reply:
Just remove the "Client" property

Reply:
Removed it.